### PR TITLE
[Korean Docs] Update docs for v12.x

### PR DESCRIPTION
**Pull request by ChangJoo-Park**

### CODE REVIEW

**Comment by ChangJoo-Park** [comment]
ping @kazupon  Can you review this PR?

File: docs/kr/configurations/custom-blocks.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.vue$/,
+        loader: 'vue',
+        options: {
+          loaders: {
+            // extract all <docs> content as raw text
```
Comment (kazupon) [inline docs/kr/configurations/custom-blocks.md]: Don't you need to translate also here?

File: docs/kr/configurations/custom-blocks.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+            'docs': ExtractTextPlugin.extract('raw-loader'),
+          }
+        }
+      }
+    ]
+  },
+  plugins: [
+    // output all docs into a single file
```
Comment (kazupon) [inline docs/kr/configurations/custom-blocks.md]: Don't you need to translate also here?

File: docs/kr/configurations/custom-blocks.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+}
+```
+
+## 런타임에서 사용할 수 있는 문서
+
+다음은 `<docs>` 사용자 정의 블록을 컴포넌트에 넣어 런타임에서 사용할 수 있는 예제입니다.
+
+#### docs-loader.js 
+
+In order for the custom block content to be injected, we'll need a custom loader:
```
Comment (kazupon) [inline docs/kr/configurations/custom-blocks.md]: need to translate here.

File: docs/kr/features/postcss.md [hunk 4]
```diff
@@ -2,54 +2,68 @@
...
 ```
 
-`postcss` 옵션은 플러그인의 배열을 제공하는 것 외에도 다음 옵션을 사용할 수 있습니다.
+`postcss` 옵션은 플러그인 배열을 제공하는 것 이외에 아래의 내용을 허용합니다.
 
-- 플러그인 배열을 반환하는 함수.
+- 플러그인 배열을 반환하는 함수
 
-- PostCSS 프로세서에 전달할 옵션을 포함하는 객체. 이것은 사용자 정의 파서/문자열 변환기에 의존하는 PostCSS 프로젝트를 사용할 때 유용합니다.
+- An object that contains options to be passed to the PostCSS processor. This is useful when you are using PostCSS projects that relies on custom parser/stringifiers:
```
Comment (kazupon) [inline docs/kr/features/postcss.md]: forget remove it?

**Comment by ChangJoo-Park** [comment]
Thanks @kazupon !! I made so many mistakes 😭 

Please check latest commit 😄

File: docs/kr/features/postcss.md (hunk 4 above)
Comment (ChangJoo-Park) [inline docs/kr/features/postcss.md]: You are right

File: docs/kr/configurations/custom-blocks.md (hunk 3 above)
Comment (ChangJoo-Park) [inline docs/kr/configurations/custom-blocks.md]: 👍

**Comment by ChangJoo-Park** [comment]
Thanks 👍 

Does it automatically apply to vue-loader.vuejs.org/kr?

**Comment by ChangJoo-Park** [comment]
additionally, https://router.vuejs.org/kr/ is not have latest version of vue-router repo.

**Comment by kazupon** [comment]
update vue-loader & vue-router docs done!
please check these. 😉

**Pull request merged**